lab-teddy/README.md

```diff
@@ -0,0 +1,12 @@
+# Project Description:
```

Comment:
Nice job with the read me so far. Though I would be more specific on the contents of your app (endpoints) instead of the dependencies it uses.